chutney/server/src/test/resources/blackbox/actions/jakarta.feature

```diff
@@ -1,172 +0,0 @@
-# language: en
```

Comment:
I think this file has not been properly deleted

Reply:
jakarta.feature is well deleted. I forgot about the campaign one. Is it ok for you ?